src/box/lua/config/source/file.lua

```diff
@@ -47,6 +44,57 @@ local function new()
     }, mt)
 end
 
+function methods.sync(self, config_module, _iconfig)
+    assert(config_module._config_file ~= nil)
+
+    local sources = {config_module._config_file}
+    local processed_sources = {}
```

Comment:
Nit: The `source` term is already used here with another meaning. I would use some other term: `path`, `config_path` or something else.

Reply:
Good point, I've renamed bunch of variables.